### ISSUE TITLE
Only updates elements that have actually changed [FEATURE REQUEST]

### ISSUE THREAD

**Issue by shacharoz** [simonoliver/UnityFigmaBridge#45]
**problem**
each time i make SYNC command, the entire prefab library is being deleted, and reimported. this is not what "synchronization" means, and its confusing.

**potential solution**
i would prefer that only the things that have changed, actually will happen.
if an object is moved >> transform will change.
if an object is added or removed >> this will reflect.
if internal structure is changed or image change >> etc

**alternatives **
plugin FigmaImporter is the other "competitor".
you have the advantage since you are creating prefabs etc.

**Additional context**
1- the huge issue here, is that each time a designer makes a small change on Figma, i am risking to change the entire unity project.
2- code components that depend on the hierarchy are not feasible. i have to use component name to connect various items, etc.

**Comment by simonoliver** [comment]
Yeah I'd love to have added this - I had a plan to implement and only apply differences and not overwrite unchanged files. Unfortunately Unity's meltdown made me switch engines so I'm primarily focussed on Unreal now so I'm not investing a lot of time investing in this project.

**Comment by shacharoz** [comment]
@simonoliver i totally understand you.
what are yoiu doing with unreal now?
how can i find your portfolio?

**Comment by simonoliver** [comment]
This is my studio - https://www.handcircus.com/ - gonna be announcing new game soon - https://www.handcircus.com/games/project-block-rancher
